cdc/api/v2/model.go

```diff
@@ -1327,3 +1327,34 @@ type OpenProtocolConfig struct {
 type DebeziumConfig struct {
 	OutputOldValue bool `json:"output_old_value"`
 }
+
+// ServiceSafePoint is the safepoint for a specific service
+// NOTE: This type is a copy of github.com/tikv/pd/pkg/storage/endpoint.ServiceSafePoint.
+// To reduce dependency tree, we do not import the api package directly.
```

Comment:
Is it necessary to reduce dependency trees? Please explain the advantages.

Reply:
We add `ExpiredTime` to display time more readable to humans.